pdns/nsecrecords.cc

```diff
@@ -109,6 +113,7 @@ NSECRecordContent::DNSRecordContent* NSECRecordContent::make(const DNSRecord &dr
       for(int bit = 0; bit < 8 ; ++bit , val>>=1)
         if(val & 1) {
           ret->d_set.insert((7-bit) + 8*(k) + 256*window);
+          ret->d_size_in_bytes += 2 + 2*sizeof(void*);
```

Comment:
Shouldn't we have the same thing for `NSEC3` records too?

Reply:
> Shouldn't we have the same thing for NSEC3 records too?

Typed it in but forgot to commit&push; now pushed. The code duplication bothers me there btw.